classes/submission/reviewer/form/ReviewerReviewStep1Form.inc.php

```diff
@@ -23,7 +23,10 @@ class ReviewerReviewStep1Form extends ReviewerReviewForm {
 	 */
 	function __construct($request, $reviewerSubmission, $reviewAssignment) {
 		parent::__construct($request, $reviewerSubmission, $reviewAssignment, 1);
-		$this->addCheck(new FormValidator($this, 'privacyConsent', 'required', 'user.profile.form.privacyConsentRequired'));
+		$context = $request->getContext();
+		if (!$reviewAssignment->getDeclined() && !$reviewAssignment->getDateConfirmed() && $context && $context->getSetting('privacyStatement')) {
```

Comment:
`$context` should always be set -- rather than giving it a pass when it's not, that should raise an error. I'd suggest assuming `$context` will be set appropriately and letting it become a "function call on non-object" if that expectation isn't met.

Reply:
(Otherwise, please go ahead! And port this to `ojs-stable-3_1_1` / `omp-stable-3_1_1` too please?)